src/main/java/net/dv8tion/jda/api/interactions/commands/Command.java

```diff
@@ -339,41 +341,41 @@ public int getId()
          * Create a Choice tuple
          *
          * @param name
-         *        The display name of this choice
+         *        The display name of this choice, must be less than 100 characters
          * @param value
          *        The integer value you receive in a command option
          */
         public Choice(@Nonnull String name, long value)
         {
-            this.name = name;
+            setName(name);
```

Comment:
Everywhere that now calls `setName` or `checkName` need to have `IllegalArgumentException` added to their docs.